### PR TITLE
Adds missing "Comment" property for ChoiceRules, Catcher and Retrier

### PR DESCRIPTION
**Pull request by alanbo**
*Issue #, if available:*
#31 

*Description of changes:*
I added Comment property to for Catch, Retry and ChoiceRule for both internal property schema and JSON Schema.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/json-schema/partial/common.json [hunk 1]
```diff
@@ -158,6 +158,11 @@
                         "type": "number",
                         "description": "A number (>= 1) which is the multiplier that increases the retry interval on each attempt. (Default: 2.0)",
                         "minimum": 1.0
+                    },
+                    "Comment": {
```
Comment (jormello) [inline src/json-schema/partial/common.json:162]: Can we reference a top-level common definition for Comment instead of repeating the definition?

Comment (jormello) [inline src/json-schema/partial/common.json:162]: Is it possible to use a ref to "#/definitions/Comment" instead of redefining it in each block?

Comment (alanbo) [reply src/json-schema/partial/common.json:160]: dibe

Comment (alanbo) [reply src/json-schema/partial/common.json:160]: done

**Pull request merged**